step_definitions/cucumber_mappings.rb

```diff
@@ -50,7 +50,7 @@ def step_file(pattern)
   end
 
   def pattern_exists?(pattern)
-    File.exist?(File.join(current_dir, step_file(pattern)))
+    File.exist?(File.join(expand_path("."), step_file(pattern)))
```

Comment:
Done the same thing in 11d6379b9096f6aa9920e2861bd0530ca7c6302b. Didn't see this was a separate commit. Sorry, you'll have to rebase skipping this commit :-/